src/main.rs

```diff
@@ -18,34 +20,48 @@ fn main() {
     loop {
         println!("Please enter a frontend problem id, or \"random\" to generate a random one.");
```

Comment:
Shall we add a message to explain the `solve $i` command?

Reply:
fixed in https://github.com/aylei/leetcode-rust/pull/18/commits/0d72e628b6c6b11cc5edca9725e3d505fcc7c355